Gemfile

```diff
@@ -2,5 +2,7 @@ source('https://rubygems.org')
 
 gemspec
 
+gem 'abbrev'
```

Comment:
Hmm is this added for any reason? Couldn't find anything that uses it.

Reply:
Hmm what's the error? Seems weird it's requiring this dependency without using it... If it's a dependency of a dependency, it shouldn't be part of the `Gemfile`, but just of the `Gemfile.lock` I think...

Reply:
```
warning: abbrev was loaded from the standard library, but is not part of the default gems starting from Ruby 3.4.0.
You can add abbrev to your Gemfile or gemspec to silence this warning.
```

Reply:
ohh interesting... I wonder if we had this warning before... I think this is probably fine, but I wonder if we need to add it to the gemspec as well... that would propagate the dependency to users of the plugin... But well, since it's a warning, i think we can do this for now and revisit later if needed 👍